src/main/java/org/sciborgs1155/robot/shooter/Shooter.java

```diff
@@ -0,0 +1,72 @@
+package org.sciborgs1155.robot.shooter;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import java.util.function.DoubleSupplier;
+import monologue.Logged;
+import monologue.Monologue.LogBoth;
+import org.sciborgs1155.robot.Robot;
+
+public class Shooter extends SubsystemBase implements Logged, AutoCloseable {
+  private final FlywheelIO flywheel;
+
+  @LogBoth
+  private final PIDController pidController =
+      new PIDController(ShooterConstants.kp, ShooterConstants.ki, ShooterConstants.kd);
+      
+
+  private final SimpleMotorFeedforward feedForward =
+      new SimpleMotorFeedforward(
+          ShooterConstants.kSVolts, ShooterConstants.kVVoltSecondsPerRotation);
+
+  @LogBoth public double target;
+
+  @LogBoth
+  public boolean isAtGoal() {
+    return pidController.atSetpoint();
+  }
+
+  public Shooter(FlywheelIO flywheelIOtype) {
+    flywheel =
+        flywheelIOtype; // in theory, this sets a final flywheel to the correct type of object thing
+    // (like real/fake/no flywheels)
+    setDefaultCommand(shoot(() -> 0.1));
```

Comment:
why is this your default? shouldn't the default be to stop?

Reply:
Idk asa said it should always be moving a little